src/security-council-mgmt/SecurityCouncilMemberSyncAction.sol

```diff
@@ -34,8 +37,12 @@ contract SecurityCouncilMemberSyncAction is ActionExecutionRecord {
         // Additionally a retryable ticket could be used to execute the update, and since tickets
         // expire if not executed after some time, then allowing updates to be skipped means that the
         // system will not be blocked if a retryable ticket is expires
-        if (_nonce <= getUpdateNonce(_securityCouncil)) {
-            revert UpdateNonceTooLow({securityCouncil: _securityCouncil, nonce: _nonce});
+        uint256 updateNonce = getUpdateNonce(_securityCouncil);
+        if (_nonce <= updateNonce) {
+            // when nonce is too now, we simply return, we don't revert.
+            // this way, rexecution for outbox execution messages or retryable tickets can't be attempted
```

Comment:
maybe we can make this clearer by saying:
```
// this way an out of date execution will actual execute, rather than remaining in an unexecuted state forever"
```